README.md

```diff
@@ -92,7 +92,7 @@ Once you're SSH'd into the jumpbox, follow these steps for deploy.
        $ cd ansible
        $ pipenv run ansible-playbook site.yml
 
-*Note:* Until [this issue](https://github.com/GSA/datagov-deploy/issues/1552) gets resolved, the command to run playbooks in sandbox is `pipenv run ansible-playbook site.yml --inventory inventories/ci --skip-tags database --limit '!inventory-2-8-web1tf.internal.ci.datagov.us'`.
+*Note:* Until [1552](https://github.com/GSA/datagov-deploy/issues/1552) and [1061](https://github.com/GSA/datagov-deploy/issues/1061) gets resolved, you should not run inventory in sandbox. The command is `pipenv run ansible-playbook site.yml --inventory inventories/ci --skip-tags database --limit 'all!inventory-web`.
```

Comment:
We should use  the [command on the wiki](https://github.com/GSA/datagov-deploy/wiki/Releases#deploy-to-sandbox). We don't want to exclude inventory-web, we just want to skip the inventory.yml playbook. It's important that we still run the common.yml playbook on inventory-web hosts.

```suggestion
*Note:* Until [1552](https://github.com/GSA/datagov-deploy/issues/1552) and [1061](https://github.com/GSA/datagov-deploy/issues/1061) gets resolved, you should not run inventory in sandbox.

    $ ansible-playbook --inventory inventories/ci --limit '!inventory-2-8-web1tf.internal.ci.datagov.us' common.yml dashboard.yml wordpress.yml catalog.yml 
```